specification/ws/src/main/scripts/org/kaazing/specification/ws/limits/should.fail.binary.payload.length.131073/handshake.request.and.frame.rpt

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+property key ${ws:handshakeKey()}
+property writeMask ${ws:randomBytes(4)}
+property client131073 ${ws:randomBytes(131073)}
+
+connect http://localhost:8081/echo
```

Comment:
Please make all your scripts use http://localhost:8080/echo to follow the convention in all of the other ws specification scripts. We can solve the problem of needing different Gateway configurations by using different test classes (see my comments on your other PR).